### DISCUSSION TITLE
How to create custom marker in marker symbol?

### DISCUSSION THREAD

**Discussion by jesuelab** [mapsforge/vtm#833] [Q&A] [answered]
How to create custom marker in marker symbol? like marker icon with name bottom

**Comment by devemux86** [comment, marked as answer]
Create a bitmap with everything you need, icon, text, etc. and then set it as marker.

For example see [AtlasMultiTextureActivity](https://github.com/mapsforge/vtm/blob/master/vtm-android-example/src/org/oscim/android/test/AtlasMultiTextureActivity.java).
